ColabTool/LinkPost/index.jsx

```diff
@@ -0,0 +1,122 @@
+import React, { PropTypes } from 'react';
+import {
+  Card,
+  Image,
+  Link,
+  LinkifiedText,
+  Text,
+} from '@bufferapp/components';
+import style from './style.css';
+import Post from '../Shared/Post';
+
+const LinkPost = ({
+  isConfirmingDelete,
+  isDeleting,
+  isWorking,
+  imageSrc,
+  links,
+  linkAttachment,
+  manager,
+  onApproveClick,
+  onCancelConfirmClick,
+  onDeleteClick,
+  onDeleteConfirmClick,
+  onEditClick,
+  profile,
+  text,
+}) => {
+  const children = (
+    <div className={style['post-content']}>
+      <div className={style['post-content-text']}>
+        <LinkifiedText
+          links={links}
+        >
+          {text}
+        </LinkifiedText>
+      </div>
+      <div className={style['link-attachment-container']}>
+        <Link href={linkAttachment.url} unstyled>
```

Comment:
Saw this when in full screen mode, the link doesn't take up the full width

![screen shot 2017-02-17 at 9 28 19 am](https://cloud.githubusercontent.com/assets/1388079/23071317/825f753c-f4f3-11e6-94ec-44ea37c3936d.png)

I think it would be good to have a `fillContainer` prop on the link. Like we did with the button 😄